index.js

```diff
@@ -361,15 +386,21 @@ function forEachTranslation(json, callback, prefix = '') {
 
 // Find all translation keys that appear in translation map A, but not
 // in translation map B
-function findDifferenceInTranslations(mapA, mapB, whitelist) {
+function findDifferenceInTranslations(mapA, mapB, whitelist, usedWhitelistEntries) {
   let missingTranslations = new Map();
 
   for (let [key, files] of mapA) {
     const keyTrimmed = key.trim();
     const isKeyMissing = !mapB.has(keyTrimmed);
-    const isKeyAllowed = !whitelist.some(regex => regex.test(keyTrimmed));
 
-    if (isKeyMissing && isKeyAllowed) {
+    if (!isKeyMissing) continue;
```

Comment:
Little performance improvement here; if the key wasn't missing, it would still be looked up in the whitelist, which is useless.